README.md

```diff
@@ -1,2 +1,59 @@
-# outreach
-A Node.js community growth and support initiative
+<p align="center">
+  <br>
+  <a href="https://nodejs.dev">
+    <img src=".github/images/nodejslogo.png" width="200"/>
+  </a>
+</p>
+
+<h1 align="center"><a href="https://node.js.dev"> Outreach Initiative</a></h1>
+
+<p align="center">The Outreach Initiative is aimed to provide help, guidance, and spotlight to the Node.js community.</p>
+
+<p align="center">
+  <a title="MIT License" href="LICENSE">
+    <img src="https://img.shields.io/github/license/gridsome/gridsome.svg?style=flat-square&label=License&colorB=6cc24a">
+  </a>
+  <a title="Follow on Twitter" href="https://twitter.com/Node.js">
```

Comment:
Small thing, twitter link is broken.

```suggestion
  <a title="Follow on Twitter" href="https://twitter.com/Nodejs">
```

Reply:
Oh, thanks a lot for this. I did a bulk/replace and didn't think of this.